src/js/themes/media-theme-micro.js

```diff
@@ -0,0 +1,538 @@
+/* 
+<media-theme-micro>
+  <video
+    slot="media"
+    src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"
+  ></video>
+</media-theme-micro>
+*/
+
+import { window, document } from '../utils/server-safe-globals.js';
+import { MediaThemeElement } from '../media-theme-element.js';
+
+const html = (raw, ...keys) => String.raw({ raw }, ...keys);
+
+const template = document.createElement('template');
+template.innerHTML = html`
+<style>
+  media-captions-button:not(:is([media-captions-list], [media-subtitles-list])) {
```

Comment:
suggestion(non-blocking):
```suggestion
  media-captions-button:not(:is(media-captions-button[media-captions-list], media-captions-button[media-subtitles-list])) {
```

Might be better to repeat the `media-captions-button` selector. Otherwise, the `:is` will try to look through *all* elements.